.github/ISSUE_TEMPLATE/bug-issues.md

```diff
@@ -0,0 +1,15 @@
+---
+name: Bug
+about: Reporting bugs and problems in Eclipse Codewind
```

Comment:
Should we just refer to it as `Codewind` to avoid any possible confusion with the Eclipse IDE tools?